bin/generate_prototype_data.rb

```diff
@@ -27,8 +28,13 @@ def run
 
       puts "Getting format and display_type for each content item..."
       content_items.each do |content_item|
-        search_endpoint = URI "#{hostname}/api/search.json?filter_link=#{content_item["base_path"]}"
         puts "-- Fetching search data for #{content_item["base_path"]}"
+
+        content_item_endpoint = JSON.parse(Net::HTTP.get(URI(content_item["api_url"])))
```

Comment:
The messages printed to STDOUT don't match what's happening anymore. They're useful for quickly spotting where something broke. I'd add another `puts` in for your timestamp code and reorder these lines so the messages line up with the code being executed.